{{cookiecutter.github_project_name}}/package.json

```diff
@@ -29,12 +29,12 @@
     "url": "https://github.com/{{ cookiecutter.github_organization_name }}/{{ cookiecutter.github_project_name }}"
   },
   "scripts": {
-    "build": "npm run build:lib && npm run build:nbextension",
-    "build:labextension": "npm run clean:labextension && mkdirp {{ cookiecutter.python_package_name }}/labextension && cd {{ cookiecutter.python_package_name }}/labextension && npm pack ../..",
+    "build": "npm run build:lib && npm run build:nbextension && npm run build:labextension",
```

Comment:
Is it actually a good idea to change this line? I personally develop mostly in Jupyter Notebook, and having `npm run build` trigger a `jupyter labextension build .` is not very friendly with this workflow, it will simply fail in envs where JupyterLab is not installed.

Reply:
Also, is it actually supposed to work with JupyterLab 2? It seems to fail with `Please supply at least one subcommand: check, disable, enable, install, link, list, uninstall, unlink, update`